### ISSUE TITLE
UWP App Crash when clicking on certain DOM nodes: Object doesn't support property or method 'hasAttribute'

### ISSUE THREAD

**Issue by cmaart** [driftyco/ionic#10111]
**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
Click on various (?) dom elements sometimes leads to an error in Edge browser (this affects UWP apps).
In our case it's a svg icon that is embedded like this:
```
<a>
 <svg>
  <use xlink:href="icons-nav.svg#comment"></use>
 </svg>
 <span>Comments</span>
</a>

```
**Expected behavior:**
No error is thrown.

**Steps to reproduce:**
I couldn't get a plunkr quickly working with embedded svg.
In our case: multiple clicks (or taps) on the svg icon and the UWP app crashes immediately.

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
insert the output from ionic info here
 ordova CLI: 6.4.0                  
Ionic Framework Version: 2.0.0-rc.5 
Ionic CLI Version: 2.2.1            
Ionic App Lib Version: 2.2.0        
Ionic App Scripts Version: 1.0.0    
ios-deploy version: Not installed   
ios-sim version: Not installed      
OS: Windows 10                      
Node Version: v7.3.0                
Xcode version: Not installed        
```

In this screenshot you can see the case where on the affected line of code the variable ele is an object that doesn't have the hasAttribute method (the watch window on the right shows this).
On the line where it tries to call hasAttribute it's gonna fail with:
`Object doesn't support property or method 'hasAttribute'`

![image](https://cloud.githubusercontent.com/assets/4008452/22149539/abd72f9e-df14-11e6-90f0-a08955c21687.png)

**Comment by cmaart** [comment]
In the case where it crashes, Edge shows for the ele variable: 
`[object SVGElementInstance]`
and in the cases where the click works fine:
`[object SVGSVGElement]`

**Comment by cmaart** [comment]
This issue was fixed by commit: 9168bc137b9c98536bc6a087d5977145e75654f2
